applicationinsights.ts

```diff
@@ -22,6 +23,7 @@ class ApplicationInsights {
     * The default client.
     */
     public static client: Client;
+    public static contracts = Contracts;
```

Comment:
Do you think this should be capitalized? I know all of the other fields are lowercase, but this isn't a field so much as exposing a namespace. @KamilSzostak does JS SDK have any precedence on this to follow?